scripts/RNA_analysis/make_complexity_plots.py

```diff
@@ -46,11 +48,19 @@
 #SBATCH {5}
 
 module load bioinfo-tools
+module load {6}
```

Comment:
Again, small thing, but a few comments here would be nice :) Especially with these numbered arguments. Would make it a lot easier to understand what's going on..

```bash
module load {6} # preseq
module load {3} # samtools
[ ... ]

# Six arguments given - must mean that <single end> was specified
if len(sys.argv) == 6:
```